tools/report-converter/README.md

```diff
@@ -123,6 +124,16 @@ clang++ -fsanitize=thread -g tsan.cpp
 report-converter -t tsan -o ./tsan_results tsan.output
 ```
 
+## [Cppcheck](http://cppcheck.sourceforge.net/)
+```sh
+# Run Cppcheck.
+cppcheck --project=compile_command.json --plist-output=./cppcheck_reports
```

Comment:
I would mention here that you have to create a report directory first. Do we want to create almost the same documentation at two places?https://github.com/Ericsson/codechecker/blob/1d855af1d005b45a253637a6fa23cf01fbae9401/docs/cppcheck.md

Reply:
I moved the Cppcheck documentation under the `report-converter` tool.